### PR TITLE
Initial Build v0.3.9

### PR DESCRIPTION
**Pull request by ViridianMelody**
Jira issue: https://anaconda.atlassian.net/browse/PKG-750
Upstream Repo: https://github.com/catherinedevlin/ipython-sql

Very minimal changes from conda-forge.

### CODE REVIEW

File: recipe/github__sodre__ipython-sql__py27-setup.patch [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
---- a/setup.py
-+++ b/setup.py
-@@ -1,5 +1,9 @@
- from setuptools import setup, find_packages
- import os
-+import sys
-+
-+if sys.version_info[0] == 2:
-+    from io import open
-
```
Comment (ryanskeith) [inline recipe/github__sodre__ipython-sql__py27-setup.patch:12]: I don't think this is required.  We should not be supporting Python 2.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -30,19 +30,25 @@ requirements:
...
+  requires:
+    - pip
+  commands:
+    - pip check
   imports:
     - sql
 
 about:
-  home: https://pypi.python.org/pypi/ipython-sql
+  home: https://pypi.org/project/ipython-sql/
```
Comment (ryanskeith) [inline recipe/meta.yaml]: Pypi, should not be where we are pointing as home.  Pypi points back to the dev url which is better.

Comment (ryanskeith) [inline recipe/meta.yaml]:
```suggestion
  home: https://github.com/catherinedevlin/ipython-sql
```

**Review by ryanskeith — CHANGES_REQUESTED** [review]
Overall good.  I think adding a patch to support Python 2 is a bad idea as we will not support it.

File: recipe/github__sodre__ipython-sql__py27-setup.patch (hunk 1 above)
Comment (ViridianMelody) [reply recipe/github__sodre__ipython-sql__py27-setup.patch:12]: This change is to remove the existing patch.

Comment (ryanskeith) [reply recipe/github__sodre__ipython-sql__py27-setup.patch:12]: Apologies, it seemed like it was adding it in.

**Review by ryanskeith — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 3]
```diff
@@ -6,22 +6,22 @@ package:
...
+  skip: True  # [py<36]
   script: 
     - python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: Missing `setuptools`

**Pull request merged**